server/portal/apps/portal_messages/models.py

```diff
@@ -41,7 +41,8 @@ class CustomMessageTemplate(models.Model):
 
     COMPONENTS = [('DASHBOARD', 'Dashboard'), ('DATA', 'Data Files'),
                   ('APPLICATIONS', 'Applications'), ('ALLOCATIONS', 'Allocations'),
-                  ('HISTORY', 'History'), ('UI', 'UI'), ('ACCOUNT', 'Account')]
```

Comment:
We may need to add a tuple for the Submissions component once that branch is merged in.